index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+
+    <body>
+        <img id="unboardingimg" src=" ./assets/image/unboarding.svg " alt=" ">
+        <div class="container">
+            <p id="managetext">Manage Your Task</p>
+
+            <input id="name" onselect="fun(this)" type="text" name=" " placeholder="Enter your name ">
```

Comment:
I think it needs to be more organized, the title, input and button to be in the same level and direction


<img width="327" alt="Screen Shot 2021-11-11 at 12 23 32 PM" src="https://user-images.githubusercontent.com/47992412/141282167-8aa29485-1309-4941-ad7e-7b7ea15a9f5b.png">